jobs/github-event/installation_repositories/added.js

```diff
@@ -23,8 +26,34 @@ module.exports = async function ({ installation, repositories_added }) {
 
   const repositories = await Promise.mapSeries(repositories_added, doc => {
     const [owner, repo] = doc.full_name.split('/')
-    return GithubQueue(installation.id).read(github =>
-      github.repos.get({ owner, repo }))
+    return GithubQueue(installation.id).read(github => {
+      return promiseRetry((retry, number) => {
+        /*
+          if we get a 404 here, log, and try again a few times.
+          we’re doing the retry here so we can job-specific logs
+          and to keep the retry logic in lib/github.js simple
+        */
+        return github.repos.get({ owner, repo })
+        .catch(error => {
+          if (error.code === 404) {
+            if (number === max404Retries) {
+              // ignore and log failure here
+              // console.log('repo not found ' + number + 'th try: giving up')
```

Comment:
Suuuuuuuper nitpicky, but maybe log `repo not found on attempt #${number}` so we don’t have `1th` and `2th` try :D